src/icon/icon.jsx

```diff
@@ -31,7 +32,12 @@ const Icon = ({
   // Icon from icomoon have different path (up to 13) to handle colors
   // We inject all of them
   const iconComponent = (
-    <i name={iconCode} className={iconCode} title={title}>
+    <i
+      name={iconCode}
+      className={iconCode}
+      title={title}
+      aria-hidden="true"
```

Comment:
means: always hidden for Assistance Technologies (screen reader)

Reply:
Why don't you do: `aria-hidden={!alt || !labelLeft || !labelRight}` ?